config/routes.rb

```diff
@@ -1,12 +1,19 @@
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  # Defines the root path route ("/")
-  # root "articles#index"
-  
+# Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+# Defines the root path route ("/")
+# root "articles#index"
+
 Rails.application.routes.draw do
+  # devise_for :users
   root "users#index"
 
   resources :users, ony: [:index, :show] do 
-    resources :posts,only: [:index, :show]
+    resources :posts,only: [:index, :show, :new, :create] do 
+      resources :comments, only: [:create]
```

Comment:
- [ ] Good job so far but  I noticed that one comment route is missing as mentioned by previous reviewer please add routes to handle the Comment Controller actions you should have something like the example below :arrow_down:  . Please fix it in order to make your project works as expected :+1: 

```
/// Already implemented    
POST   /posts/:post_id/comments(.:format)     comments#create

/// Missing route
 GET   /posts/:post_id/comments/new(.:format)     comments#new

```